server/subscriptions.go

```diff
@@ -108,6 +117,9 @@ func (p *Plugin) Subscribe(ctx context.Context, githubClient *github.Client, use
 		CreatorID:  userId,
 		Features:   features,
 		Repository: fullNameFromOwnerAndRepo(owner, repo),
+		Flags: SubscriptionFlags{
+			ExcludeOrgMembers: containsValue(flags, "exclude-org-member"),
```

Comment:
I think it'll be slightly better if we move this parsing logic to https://github.com/mattermost/mattermost-plugin-github/pull/154/files#diff-7278f3371df5ef589578d375bde66febR126 and just pass the `SubscriptionFlags` struct around.

The benefit of that will be that we'll have all the `SubscriptionFlags` parsing logic in one spot. With the current approach if we decided to add a new flag you'll have to update in two spots.

Reply:
Thanks for your comments Ali! That was a doubt I had while implementing it. I eventually decided to place it here just because I wasn't sure if this struct should be created within its own domain (within subscriptions.go in this case)

Reply:
@enolal826 
Fair point. Ideally I think what we should be doing is inside `commands.go` we should be creating a `Subscription` struct and passing that in, however that's a much bigger change. 

Also, after your comment I'm thinking maybe we should just put `ExcludeOrgMembers` directly under `Subscription` and get rid of `SubscriptionFlags` and just pass in the list of flags? what do you think?

Reply:
Not sure. I think it might be good to have the flags in a different struct (in case we add new flags it might confusing with the "Features" field), but you probably have more knowledge on the codebase, so I can update it if you prefer it

Reply:
Fair enough, in that case can we move the creation of `SubscriptionFlags` close to where we parse it?

Reply:
I've updated it. What do you think now?

Reply:
@enolal826 looks good 👌

Reply:
@enolal826 can you please resolve conflicts and merging master into your branch? thanks!

Reply:
Done, thanks!